### PR TITLE
Adding 'connect dataset' option in new dashboard being in data library section

### LINKED ISSUE #2929: There's no option to create a dataset when you click on a data library row
If you are in the datasets library view and click on a dataset I would expect to have a button to import it. 

This is thew view when no dataset is selected:  

screenshot from 2015-03-26 18 06 28

And this is the view when a dataset is selected:  

screenshot from 2015-03-26 18 06 34

/cc @saleiva

### PR DESCRIPTION
**Pull request by xavijam**
It lets user to import a data library from dashboard list view.

screen shot 2015-03-27 at 11 46 05

Fixes #2929 

REVIEWERS: @lbosque @saleiva

### CODE REVIEW

**Comment by xavijam** [comment]
retest this please

**Comment by xavijam** [comment]
retest this please

**Comment by xavijam** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2601/))

**Comment by xavijam** [comment]
No comments?

**Comment by xavijam** [comment]
Adding @javierarce

File: lib/assets/javascripts/cartodb/new_dashboard/filters_view.js [hunk 1]
```diff
@@ -273,6 +274,30 @@ module.exports = cdb.core.View.extend({
     }
   },
 
+  _importRemote: function(e) {
+    if (e) this.killEvent(e);
+
+    var selectedItems = this._selectedItems();
+    if (selectedItems.length !== 1) {
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/filters_view.js]: Shouldn't it be `selectedItems.length < 1`?

Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/filters_view.js]: Only works when there is one dataset selected. Disabled in any other case.

File: lib/assets/test/spec/cartodb/new_dashboard/filters_view.spec.js [hunk 2]
```diff
@@ -143,12 +143,52 @@ describe('new_dashboard/filters_view', function() {
       expect(this.innerHTML()).toContain('Deselect all yours');
     });
 
+    describe('create dataset', function() {
+      beforeEach(function() {
+        spyOn(this.user, 'canCreateDatasets').and.returnValue(true);
+      });
+
+      it('should be displayed when only one library dataset is selected', function() {
```
Comment (javierarce) [inline lib/assets/test/spec/cartodb/new_dashboard/filters_view.spec.js]: Wouldn't it make sense to add a specs for when there's nothing / several items are selected?

Comment (xavijam) [inline lib/assets/test/spec/cartodb/new_dashboard/filters_view.spec.js]: :+1:

File: lib/assets/javascripts/cartodb/new_dashboard/filters_view.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/filters_view.js]: Oops, sorry, you're totally right. Forget what I said :)

**Comment by javierarce** [comment]
Other than the suggestion of adding a couple of little specs, code looks fine.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2630/))

**Pull request merged**